ducktape/tests/runner_client.py

```diff
@@ -157,7 +158,7 @@ def run(self):
     def setup_test(self):
         """start services etc"""
         self.log(logging.INFO, "Setting up...")
-        self.test.setUp()
+        self.test.setup()
```

Comment:
Right, the reason there was still some uncertainty is because it was unclear exactly how the separate streams running over the same SSH connection would work, i.e. if one ultimately could end up blocking things up such that another can't make any progress. If that does appear to be a problem now, we can definitely fix it in ducktape.